src/views/partials/__footer.njk

```diff
@@ -5,28 +5,19 @@
         <h2 class="govuk-visually-hidden">Support links</h2>
         <ul class="govuk-footer__inline-list govuk-!-display-none-print" data-module="gem-track-click" data-track-links-only="" data-gem-track-click-module-started="true">
           <li class="govuk-footer__inline-list-item">
-            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/help" data-track-options="{&quot;dimension29&quot;:&quot;Help&quot;}" href="/help">Help</a>
+            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/policies" data-track-options="{&quot;dimension29&quot;:&quot;Help&quot;}" href="https://resources.companieshouse.gov.uk/serviceInformation.shtml">Policies</a>
           </li>
           <li class="govuk-footer__inline-list-item">
-            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/help/privacy-notice" data-track-options="{&quot;dimension29&quot;:&quot;Privacy&quot;}" href="/help/privacy-notice">Privacy</a>
+            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/cookies" data-track-options="{&quot;dimension29&quot;:&quot;Privacy&quot;}" href="{{ chsUrl }}/help/cookies">Cookies</a>
           </li>
           <li class="govuk-footer__inline-list-item">
-            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/help/cookies" data-track-options="{&quot;dimension29&quot;:&quot;Cookies&quot;}" href="/help/cookies">Cookies</a>
+            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/contactUs" data-track-options="{&quot;dimension29&quot;:&quot;Cookies&quot;}" href="{{ chsUrl }}/help/contact-us">Contact us</a>
           </li>
           <li class="govuk-footer__inline-list-item">
-            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/help/accessibility-statement" data-track-options="{&quot;dimension29&quot;:&quot;Accessibility statement&quot;}" href="/help/accessibility-statement">Accessibility statement</a>
+            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/accessibility-statement" data-track-options="{&quot;dimension29&quot;:&quot;Accessibility statement&quot;}" href="{{ chsUrl }}/close-a-company/accessibility-statement">Accessibility statement</a>
```

Comment:
- do I understand that the Accessibility statement needs to come out of this?  I note the comment on the story.
- also I see there's a conflict reported on this branch - need to merge down from main